main.go

```diff
@@ -25,8 +25,8 @@ import (
 	"github.com/cilium/pwru/internal/pwru"
 )
 
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc clang KProbePWRU ./bpf/kprobe_pwru.c -- -DOUTPUT_SKB -I./bpf/headers
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc clang KProbePWRUWithoutOutputSKB ./bpf/kprobe_pwru.c -- -I./bpf/headers
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc clang-12 KProbePWRU ./bpf/kprobe_pwru.c -- -DOUTPUT_SKB -I./bpf/headers
```

Comment:
I don't think we want to pin this clang binary. Can we add a symlink in the Vagrant example instead?

Reply:
Done.